jupyter_nbconvert/_version.py

```diff
@@ -1,2 +1,3 @@
-version_info = (4, 0, 0, 'dev')
+_version_extra = 'dev'
+version_info = (4, 0, 0, _version_extra)
```

Comment:
This will do the wrong thing at release time when _version_extra is empty. My plan for this was to have a release replace this with `version_info = (4, 0, 0)`, having no extra at all.

If version_extra is to be treated as a variable, it needs to be conditionally added to version_info only if it's non-empty.

If this is prompted by the _version_extra logic in conf.py, I would prefer to remove that logic, rather than change this here.